common/models/oonitarian.json

```diff
@@ -39,7 +39,7 @@
     }
 
   },
-  "hidden": ["password", "legal_name"],
+  "hidden": ["password"],
```

Comment:
I verified that this is not exposed from the public view. The ACL on the oonitarian model stays strong and the only one with READ caps is the team that enforces privacy on an application level.